website/src/pages/contact/request-batch-changes-demo.tsx

```diff
@@ -13,7 +13,6 @@ const Contact: FunctionComponent<PageProps> = props => {
         portalId: '2762526',
         formId: 'c98d6435-f0fc-4b34-8cff-cfe7633121c8',
         targetId: 'hubspotRequestBatchChangesDemo',
-        chiliPiper: true,
```

Comment:
For pages where this is removed and it was `true`, shouldn't we add the `useChiliPiper()` hook?

Reply:
Yes! Thank you